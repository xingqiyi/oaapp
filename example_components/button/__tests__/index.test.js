import React from 'react';
import { render } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';
import Button from '../index';

describe('Button', () => {
  it('renders correctly', () => {
    const wrapper = render(<Button>foo</Button>);
    expect(renderToJson(wrapper)).toMatchSnapshot();
  });
});
