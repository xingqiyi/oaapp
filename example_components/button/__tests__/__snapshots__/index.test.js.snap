// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<a
  aria-disabled="false"
  class="am-button"
  role="button"
>
  <span>
    foo
  </span>
</a>
`;
